modules/social_features/social_topic/social_topic.install

```diff
@@ -526,7 +526,7 @@ function social_topic_update_111101(): void {
  * Permissions to check:
  * - "translate topic node".
  */
-function social_topic_update_111102(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.